smt/surrogate_models/krg_based.py

```diff
@@ -136,6 +136,12 @@ def _initialize(self):
         # theta in (0,1e1], is a good choice to avoid overfitting. By standardising
         # X in R, X_norm = (X-X_mean)/X_std, then X_norm in [-1,1] if considering
         # one std intervals. This leads to theta in (0,2e1]
+        declare(
```

Comment:
Should not be in this PR (as the related code below). Moreover we can avoid to declare another option by specifying a `None` value for the `hyper_opt` option.

Reply:
I apologize, I forgot that it was in branch master